stdlib/universe/window_test.go

```diff
@@ -517,12 +514,12 @@ func TestFixedWindow_Process(t *testing.T) {
 							{Label: "_value", Type: flux.TFloat},
 						},
 						Data: [][]interface{}{
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(60*time.Second), 6.0},
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(70*time.Second), 7.0},
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(80*time.Second), 8.0},
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(90*time.Second), 9.0},
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(100*time.Second), 10.0},
-							{start + 1*execute.Time(time.Minute), start + 2*execute.Time(time.Minute), start + execute.Time(110*time.Second), 11.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(0*time.Second), 0.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(10*time.Second), 1.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(20*time.Second), 2.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(30*time.Second), 3.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(40*time.Second), 4.0},
+							{start, start + 1*values.Time(time.Minute), start + values.Time(50*time.Second), 5.0},
```

Comment:
Underlapping windows with a period that is smaller than the every now start at the beginning of the window. This was a breaking change, so tests dealing with this needed to be updated.